src/js/apps/patients/worklist/filters_app.js

```diff
@@ -19,9 +27,42 @@ export default App.extend({
     this.showFilters();
   },
   showFilters() {
+    this.showAllFiltersButtonView();
     this.showGroupsFilterView();
     this.showOwnerToggleView();
   },
+  showAllFiltersButtonView() {
+    if (this.groups.length < 2) return;
```

Comment:
when we get to the next step this check should be replaced for one that only shows if there are any custom filters.  If there are no custom filters, we don't need the sidebar whether or not there is a group filter.